src/Sdk/DTWebApi/WebApi/ActionsStepTelemetry.cs

```diff
@@ -15,6 +16,12 @@ public ActionsStepTelemetry()
             this.ErrorMessages = new List<string>();
         }
 
+        [DataMember(EmitDefaultValue = false)]
+        public string DisplayName { get; set; }
+
+        [DataMember(EmitDefaultValue = false)]
+        public int? Order { get; set; }
+
```

Comment:
I am using step telemetry to construct step results that are needed to send as part of job completed event